draft-bbs-signatures.md

```diff
@@ -298,20 +298,20 @@ Parameters:
 
 Definitions:
 
-- HKDF-Extract is as defined in [@!RFC5869], instantiated with hash H.
-- HKDF-Expand is as defined in [@!RFC5869], instantiated with hash H.
+- HKDF-Extract is as defined in [@!RFC5869], instantiated with hash function HASH.
+- HKDF-Expand is as defined in [@!RFC5869], instantiated with hash function HASH.
 - I2OSP and OS2IP are as defined in [@!RFC8017], Section 4.
 - L is the integer given by ceil((3 * ceil(log2(q))) / 16).
-- "BBS-SIG-KEYGEN-SALT-" is an ASCII string comprising 20 octets.
+- INITSALT is the octet string "0x4242532d5349472d4b455947454e2d53414c542d", obtained by taking the UTF-8 representation of the ASCII string "BBS-SIG-KEYGEN-SALT-".
```

Comment:
Thank you @christianpaquin. Just a nit. I wonder if it would be better to "enforce" the utf-8 encoding in the [Encoding of elements to be hashed](https://github.com/decentralized-identity/bbs-signature/blob/main/draft-bbs-signatures.md#encoding-of-elements-to-be-hashed) section?? Maybe by adding the following line,
```
- Strings must be encoded to octets using the UTF-8 representation and zero-extending the result to a length that is a multiple of 8 bits.
```
That way the UTF-8 representation will be used not only for the salt in keyGen but for any string that needs to be hashed (for example a DST or the Ciphersuite_ID).

Reply:
A sorry. I just saw @tplooker's [comment](https://github.com/decentralized-identity/bbs-signature/pull/123#discussion_r865265607) above with the same proposal. Sorry about that.

Reply:
As I commented above, the rest of the spec expects octet strings, making it sound like the encoding was done at the application level. If we want to make these input ASCII strings, then we should indeed specify how to transform them in this section.

Reply:
Good catch IMO I think DST should stay as an octet string as the underlying data could be any information not just a string value but we should update the CipherSuite_ID parameter to be a string and therfore its encoding would follow the rule defined above.